config/src/main/java/org/hyperledger/besu/config/BftFork.java

```diff
@@ -40,6 +40,9 @@ public class BftFork implements Fork {
   /** The constant BLOCK_PERIOD_SECONDS_KEY. */
   public static final String BLOCK_PERIOD_SECONDS_KEY = "blockperiodseconds";
 
+  /** The constant EMPTY_BLOCK_PERIOD_SECONDS_KEY. */
+  public static final String EMPTY_BLOCK_PERIOD_SECONDS_KEY = "emptyblockperiodseconds";
```

Comment:
```suggestion
  public static final String EMPTY_BLOCK_PERIOD_SECONDS_KEY = "xemptyblockperiodseconds";
```

Reply:
I think we should make this an experimental option initially, so I'll suggest changes in my review which you can just sign-off & commit if you're happy with them.

Reply:
This one willbe needed, yes.